chat_service/src/main/kotlin/ru/zveron/exception/GrpcExceptionAdvice.kt

```diff
@@ -0,0 +1,33 @@
+package ru.zveron.exception
+
+import io.grpc.Status
+import mu.KLogging
+import net.devh.boot.grpc.server.advice.GrpcAdvice
+import net.devh.boot.grpc.server.advice.GrpcExceptionHandler
+import ru.zveron.library.grpc.exception.PlatformException
+
+@GrpcAdvice
+class GrpcExceptionAdvice {
+    companion object : KLogging()
+
+
+    @GrpcExceptionHandler(PlatformException::class)
+    fun handlePlatformException(e: PlatformException): Status {
+        logger.error { "Got platform exception: ${e.message}" }
```

Comment:
давай exception класть в аргумент, чтобы оно польностью в логе приписывалось
```suggestion
        logger.error(e) { "Platform exception" }
```